html/canvas/offscreen/text/META.yml

```diff
@@ -4,6 +4,12 @@ links:
       results:
         - test: 2d.text.measure.width.space.html
         - test: 2d.text.measure.width.space.worker.html
+        - test: 2d.text.draw.baseline.hanging.html
```

Comment:
This is https://crbug.com/1066953 again, and this and the next test are named in a way that doesn't obviously match collapsing.

Reply:
Let's drop the baseline ones and keep *. space.collapse.* here too.

Reply:
Dropped:

```
        - test: 2d.text.draw.baseline.hanging.html
        - test: 2d.text.draw.baseline.ideographic.html
```